docs/rules/padding-line-between-component-options.md

```diff
@@ -0,0 +1,124 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/padding-line-between-component-options
+description: require or disallow padding lines between top-level component options
+---
+# vue/padding-line-between-component-options
+
+> require or disallow padding lines between top-level component options
+
+- :exclamation: <badge text="This rule has not been released yet." vertical="middle" type="error"> ***This rule has not been released yet.*** </badge>
+- :wrench: The `--fix` option on the [command line](https://eslint.org/docs/user-guide/command-line-interface#fixing-problems) can automatically fix some of the problems reported by this rule.
+
+## :book: Rule Details
+
+This rule enforces consistent format of empty lines between Vue component options by either adding or removing them.
+
+<eslint-code-block fix :rules="{'vue/padding-line-between-component-options': ['error', 'always']}">
+
+```vue
+<script>
+  <!-- ✓ GOOD -->
```

Comment:
The code example should not contain syntax errors (`<!--` in JavaScript syntax) or cause other fatal ESLint parsing errors (like duplicate `export default`).

See the preview deployment: https://deploy-preview-1818--eslint-plugin-vue.netlify.app/rules/padding-line-between-component-options.html

Reply:
The new commit I pushed puts every `export default` block in a separate  `<script>` tag.
Should I separate them into multiple `<eslint-code-block>` as well?

Reply:
Hmm, since the editor in the preview deployment doesn't show squiggly red underlines, you probably have to combine both ```` ```vue ```` blocks into one; or indeed add separate `<eslint-code-block>`s.

Also, please put the code examples below the individual rule options ("always", "never"), so you can see which code example is which. See e.g. `padding-line-between-blocks` docs.